### PR TITLE
Nodejs week3

### PR DESCRIPTION
**Pull request by Kositthai**
warm up  nodejs week3

### CODE REVIEW

File: nodejs/week3/app.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+const knex = require("knex")({
```
Comment (itsgauravsaxena) [inline nodejs/week3/app.js:1]: You have knex config in app.js and database.js.
Please try to import this from database.js

File: nodejs/week3/app.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    client: "mysql2",
+    connection: {
+      host: process.env.DB_HOST || "127.0.0.1",
+      port: process.env.DB_PORT || 3306,
+      user: process.env.DB_USER || "root",
+      password: process.env.DB_PASSWORD || "123456789",
```
Comment (itsgauravsaxena) [inline nodejs/week3/app.js:7]: And password should come from env file. Its a good practise

**Review by itsgauravsaxena — CHANGES_REQUESTED** [review]

File: nodejs/week3/app.js (hunk 1 above)
Comment (Kositthai) [reply nodejs/week3/app.js]: Hi @itsgauravsaxena,  

Thank you so much for your time reviewing my homework! 
I have fixed my code as per your suggestions

Pls let me know if I need to adjust something else or if I can merge it.

**Review by itsgauravsaxena — APPROVED** [review]

**Comment by Kositthai** [comment]
Thank you so much for your time! :)

**Pull request merged**